snuba/web/db_query.py

```diff
@@ -936,7 +936,34 @@ def _apply_allocation_policies_quota(
         stats["quota_allowance"]["summary"] = summary
 
         if not can_run:
+            metrics.increment(
+                "rejected_query",
+                tags={
+                    "referrer": str(
+                        attribution_info.tenant_ids.get("referrer", "no_referrer")
```

Comment:
referrer seems kind of high-cardinality as a metric tag. do we use it elsewhere?

Reply:
Yes: https://github.com/getsentry/snuba/blob/74e80f24835f42707d5b1b23ca3c74949957e8a0/snuba/query/allocation_policies/__init__.py#L791

side note: `db_request_rejected` and `db_request_throttled` are the metrics that I mentioned in the PR description that can be counted multiple times for a single query

Reply:
Now that I think about it, since `db_request_rejected` and `db_request_throttled` already has the referrer tag, I don't need them here